distributed/http/templates/worker-table.html

```diff
@@ -20,10 +20,10 @@
             <td><a href="../worker/{{ url_escape(ws.address) }}.html">{{ws.address}}</a></td>
             <td> {{ ws.name if ws.name is not None else "" }} </td>
             <td> {{ ws.nthreads }} </td>
-            <td> {{ format_bytes(ws.memory_limit) if ws.memory_limit is not None else "" }} </td>
+            <td data-sort="{{ws.memory_limit}}"> {{ format_bytes(ws.memory_limit) if ws.memory_limit is not None else "" }} </td>
             <td> <progress class="progress" value="{{ ws.metrics['memory'] }}" max="{{ ws.memory_limit }}"></progress>
             </td>
-            <td> {{ format_time(ws.occupancy) }} </td>
+            <td data-sort={{ ws.occupancy }}> {{ format_time(ws.occupancy) }} </td>
```

Comment:
This fixes a couple of failing tests we're seeing in CI

```suggestion
            <td data-sort="{{ ws.occupancy }}"> {{ format_time(ws.occupancy) }} </td>
```